Update WMI check to include full sample (#9325)
DataDog/integrations-core: wmi_check/README.md

- Metric name as it appears in Datadog.
- The metric type.

> Another sample configuration can be found below, which will populate many more metrics on a Windows 2012 server.

Reviewer: ```suggestion
The following sample configuration populates many more metrics on a Windows 2012 server.
```
